### PR TITLE
Add `recipe_id` opt to bake command

### PR DESCRIPTION
**Pull request by cisaacstern**
👋 @yuvipanda, does this look like the right way to run only a specified recipe from a feedstock? This seems to work as intended on my local installation, but I haven't added any tests yet.

### CODE REVIEW

File: pangeo_forge_runner/commands/bake.py [hunk 1]
```diff
@@ -55,6 +55,16 @@ class Bake(BaseCommand):
         """,
     )
 
+    recipe_id = Unicode(
+        default_value="",
```
Comment (yuvipanda) [inline pangeo_forge_runner/commands/bake.py:59]: Let's set the default to `None` insteadd?

**Review by yuvipanda — CHANGES_REQUESTED** [review]
LGTM but minor suggested change

File: pangeo_forge_runner/commands/bake.py (hunk 1 above)
Comment (cisaacstern) [reply pangeo_forge_runner/commands/bake.py:59]: Yes that's better. Will fix now.

**Comment by yuvipanda** [comment]
@cisaacstern you probably need to add a test here

**Comment by cisaacstern** [comment]
Thanks for looking at this, Yuvi! I'll add a test within a day or two and ping you here when it's ready for re-review.

File: pangeo_forge_runner/commands/bake.py (hunk 1 above)
Comment (cisaacstern) [reply pangeo_forge_runner/commands/bake.py]: fixed in [ba13d4d](https://github.com/yuvipanda/pangeo-forge-runner/pull/14/commits/ba13d4d60f7d4d37aaea80660456979940ffa1bd)

**Comment by yuvipanda** [comment]
Yep, happy to merge after a test!

**Comment by cisaacstern** [comment]
👋 @yuvipanda, IMHO this is good to go. Happy to adjust if you see anything I've overlooked.

**Pull request merged**